src/Extensions/Logging/EmptyMachineInformation.cs

```diff
@@ -0,0 +1,116 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Diagnostics;
+using System.Net;
+using System.Net.Sockets;
+using System.Reflection;
+
+namespace Microsoft.Omex.Extensions.Logging
+{
+	/// <summary>
+	/// Base class for machine information
+	/// </summary>
+	public class EmptyMachineInformation : IMachineInformation
+	{
+		/// <summary>
+		/// Create instance of machine information, please keep this initialization as lightweight as possible
```

Comment:
>please keep this initialization as lightweight as possible [](start = 46, length = 58)

I think `<remarks>` tags would be more appropriate for this

Reply:
Moved, thanks